frontend/src/features/videos/VideoList.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 
 import { Typography, Box } from '@mui/material';
 import VideoCard from '../videos/VideoCard';
-import { CompareNowAction, AddToRateLaterList } from 'src/utils/action';
+import { CompareNowAction, AddToRateLaterList, AddDetailedConsultation } from 'src/utils/action';
```

Comment:
There are a few checks for the code style that you can automatically fix with 

`docker exec tournesol-dev-front yarn lint:fix`